curvefs/src/metaserver/s3compact_wq_impl.cpp

```diff
@@ -60,9 +60,9 @@ void S3CompactWorkQueueImpl::Enqueue(std::shared_ptr<InodeStorage> inodeStorage,
     }
     compactingInodes_.push_back(inodeKey);
     CopysetNodeWrapper copysetNodeWrapper(copysetNode);
-    auto task =
-        std::bind(&S3CompactWorkQueueImpl::CompactChunks, this, inodeStorage,
-                  std::ref(inodeKey), std::ref(pinfo), &copysetNodeWrapper);
+    auto task = std::bind(&S3CompactWorkQueueImpl::CompactChunks, this,
+                          inodeStorage, std::ref(inodeKey), std::ref(pinfo),
```

Comment:
inodeKey and pinfo are local variables, and you pass reference to task, so it will segment fault when executing task